dgl_sparse/include/sparse/sddmm.h

```diff
@@ -14,10 +14,15 @@ namespace sparse {
 
 /**
  * @brief Perform a sampled matrix multiplication of a sparse matrix and two
- * dense matrices. It calculates `(mat1 @ mat2) * sparse_mat`. If the sparse
- * matrix has shape (n, m), `mat1` and `mat2` must have shapes of `(n, k)` and
- * `(k, m)` or `(n,)` and `(m,)` respectively. And the returned tensor has shape
- * `(sparse_matrix->nnz(),)`.
+ * dense matrices. It calculates `(mat1 @ mat2) * sparse_mat`. The SDDMM can be
```

Comment:
Maybe use `sparse_mat * (mat1 @ mat2)` because the shape notations you used below is in that order.